packages/datetime/src/dateRangePicker.tsx

```diff
@@ -98,6 +98,12 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
      */
     onHoverChange?: (hoveredDates: DateRange, hoveredDay: Date, hoveredBoundary: Boundary) => void;
 
+    /**
+     * Called when the `shortcuts` props is enabled and the user changes the shortcut.
+     * When triggered, it will pass the selected `shortcut` and its index which is optional.
```

Comment:
```suggestion
```

no need for this line, it's redundant with the typedef

Reply:
Done